src/mds/MDCache.h

```diff
@@ -1530,6 +1530,7 @@ class MDCache {
 
   DecayCounter quiesce_counter;
   uint64_t quiesce_threshold;
+  bool quiesce_recall;
```

Comment:
I'd prefer the full name here because `quiesce_recall` without the context sounds ambiguous to me